### DISCUSSION TITLE
[疑似]只能使用从jsdelivr下载的脚本

### DISCUSSION THREAD

**Discussion by Vardariotai** [the1812/Bilibili-Evolved#3712] [综合讨论 / General]
chrome版本 94.0.4606.81
今天推送了github源的v2.4.1更新，安装之后脚本失效。后使用jsDelivr源重新安装，脚本恢复正常工作。疑似因为旧版本chrome不兼容？

**Comment by the1812** [comment]
![image](https://user-images.githubusercontent.com/26504152/193379598-dc4f934a-4a31-4fdb-9ad0-78c660d9906f.png)
